src/main/java/com/google/ehub/data/ProfileDatastore.java

```diff
@@ -0,0 +1,54 @@
+package com.google.ehub.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.sps.servlets.LoginServlet;
+import java.io.IOException;
+
+/**Manages the User Profiles stored in Datastore.**/
```

Comment:
Nit: some spacing issues
`/** Manages the User Profiles stored in Datastore. **/`

Reply:
Spacing is still a little off here

Reply:
I will fix the spacing.